src/Core/Framework/DataAbstractionLayer/Dbal/Common/RepositoryIterator.php

```diff
@@ -23,6 +23,8 @@ class RepositoryIterator
      * @var Context
      */
     private $context;
+    
+    private $endReached = false;
```

Comment:
annotation is missing

Reply:
Is it? The initial value should be enough to define the type for analysis tools, IDEs and humans. Or is it some guide that one should follow for your code?

Reply:
I will not complain, if PHPStan does not complain :smile:

Reply:
Uhm phpstan in my projects does not complain about something like that. But I do php7.4 sooo...

Reply:
the github action will fail, so no worries :wink: 

but first, fix the code style :smile: as @pweyck already said, this fails

Reply:
I made some code feng shui and freshened up the branch